1.8/administration/installing/haproxy-adminrouter.md

```diff
@@ -1,10 +1,10 @@
 ---
-post_title: Configuring HAProxy in Front of Admin Router
-nav_title: HAProxy in Front of Admin Router
-menu_order: 3.3
+post_title: Configuring a Custom Certificate
```

Comment:
i think this is a bit misleading... maybe: Custom certificate configuration with HAProxy and AdminRouter